Hotel/ViewModel/AddGuestViewModel.cs

```diff
@@ -90,7 +90,8 @@ public string ICEPhoneNumber
 
         #endregion Properties
 
-        public GuestDetailViewModel(ICommand guestCommand, Action cancelAction, Guest currentGuestData, Action afterSubmitAction)
+
+        public AddGuestViewModel(ICommand guestCommand, Action cancelAction, Guest currentGuestData, Action afterSubmitAction)
```

Comment:
naam komt niet overeen met wat dit viewmodel doet:
wat hij doet is namelijk afhankelijk van de aftersubmitaction die hij meekrijgt. GuestDetailViewModel is dus veel juister qua naamgeving